tests/unit/test_proxy_manager.py

```
@@ -64,6 +66,80 @@ def test_project_dir_derives_port(self):
+        alias_project.symlink_to(real_project, target_is_directory=True)
+
+        result_real = proxy_mgr.get_proxy_config({}, project_dir=str(real_project))
+        result_alias = proxy_mgr.get_proxy_config({}, project_dir=str(alias_project))
+
+        assert result_real["port"] == result_alias["port"]
+
+
+class TestBuildProxyUrl:
+    def test_claude_uses_sse(self):
+        url = proxy_mgr.build_proxy_url(
+            "claude", {"proxy": {"port": 8792, "port_range": 0}}, "/tmp"
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)